packages/tupaia-web/src/views/ProjectsModal.tsx

```diff
@@ -29,19 +28,22 @@ const Wrapper = styled.div`
   display: flex;
   flex-direction: column;
   padding: 0.2rem 0 0;
-  width: 54rem;
+  width: 65rem;
   max-width: 100%;
+  text-align: left;
 `;
 
 const TagLine = styled.p`
-  margin: 0.625rem 0 2.5rem;
+  margin: 0.5rem 0.4rem 1.5rem;
+  width: 40%;
+  font-weight: 300;
 `;
 
 const ProjectsGrid = styled.div`
   display: grid;
   grid-template-columns: repeat(3, 1fr);
-  gap: 1rem;
-  margin: 1.5rem 0;
+  gap: 2rem;
+  margin: 1.4rem 0;
```

Comment:
I think that there needs to be more responsive styles added since the grid doesn't space out properly on tablet size devices. Maybe it should stack to 2 columns at some size?